config.php.template

```diff
@@ -1,7 +1,20 @@
 <?php
-//This is config file, please be careful
+/**************************************************************************************/
+//                      SERVER STATUS CONFIG FILE TEMPLATE
+//                THIS FILE IS FOR CREATING YOUR CONFIG MANUALLY
+//                     !!!!!!!IMPORTANT NOTE!!!!!!!!!
+//         You will need to create your htaccess or web.config file yourself.
+//           Please head to http(s)://yourdomain.com/create-server-config.php
```

Comment:
@yigitkeremoktay people that do not like to give write access to the root directory could also just copy the contents from the script... Or better yet, we could create a folder `server_config` with files `htaccess` (without the dot) and `web.config` that they could copy directly, without having to look inside the script and the script could just move  the files & delete the directory.

Reply:
I will be adding that info also. @Pryx it would be nice if you add that info to readme also 👍